components/ButtonWithIcon.tsx

```diff
@@ -0,0 +1,62 @@
+import { View, Text, StyleSheet, TouchableOpacity, Image } from "react-native";
+
+const ButtonWithIcon = ({
+  title,
+  onPress,
+  style,
+  active,
+  color, 
+  icon
+}: {
+  title: string;
+  onPress: () => void;
+  style?: object;
+  active: boolean;
+  color: string;
+  icon: any;
```

Comment:
Refactoring this so that you can pass `require("@/assets/images/something.png` as a prop would solve this "any" typerror